tests/css-images-3.js

```diff
@@ -18,12 +18,21 @@ export default {
 			tests: [
 				'linear-gradient(white, black)',
 				'linear-gradient(to right, white, black)',
+				'linear-gradient(0, white, black)',
 				'linear-gradient(45deg, white, black)',
 				'linear-gradient(white 50%, black)',
 				'linear-gradient(white 5px, black)',
 				'linear-gradient(white, #f06, black)',
 				'linear-gradient(currentColor, black)',
 				'linear-gradient(red -50px, white calc(-25px + 50%), blue 100%)',
+
+				'linear-gradient(red 0 0)',
```

Comment:
The syntax with two `<length-percentage>`s was [only added in css-images-4](https://drafts.csswg.org/css-images-4/#changes-20120911).

So either tests like this one should be moved there and extended by different combinations of lengths and percentages, or removed from this PR and a separate PR be created to add those tests separately.

Reply:
Thanks, done!